plugin/core/registry.py

```diff
@@ -101,11 +101,36 @@ def sessions(self, capability: Optional[str] = None) -> Generator[Session, None,
         yield from sessions_for_view(self.view, capability)
 
 
-class LspRestartClientCommand(sublime_plugin.TextCommand):
-    def run(self, edit: Any) -> None:
+class LspRestartServerCommand(LspTextCommand):
+    def run(self, edit: Any, config_name: str = None, show_quick_panel: bool = False) -> None:
```

Comment:
Where is the parameter `config_name` supposed to be used?

Reply:
Probably would be useful to simplify https://github.com/sublimelsp/LSP-intelephense/pull/63

Reply:
Ah, I understand now.